lib/zxcvbn_tester.rb

```diff
@@ -0,0 +1,10 @@
+module Zxcvbn
+  class Tester
```

Comment:
makes sense. That is where Brian initially put it and I told him to move -- sorry Brian!